package.json

```diff
@@ -20,12 +20,14 @@
     "lib"
   ],
   "main": "lib/index.js",
-  "types": "lib/index",
+  "typings": "lib/index.d.ts",
   "scripts": {
     "clean": "rimraf lib && rimraf coverage",
     "format": "yarn prettier --write '{src,__tests__}/**/*.ts'",
     "lint": "tslint -p . --fix",
     "prebuild": "yarn clean",
+    "prepare": "npm run build",
+    "prepublish": "npm run build",
```

Comment:
pls use yarn as default: `yarn build`

Reply:
@manh-vv  I know. But how to user use the npm package when they don't have yarn.